frmts/vrt/vrtprocesseddataset.cpp

```diff
@@ -713,7 +716,8 @@ bool VRTProcessedDataset::ParseStep(const CPLXMLNode *psStep, bool bIsFinalStep,
         for (int i = 1; i <= m_poSrcDS->GetRasterCount(); ++i)
         {
             int bHasVal = false;
-            const double dfVal = GetRasterBand(i)->GetOffset(&bHasVal);
+            const double dfVal =
```

Comment:
Pretty sure this was a typo in the original; `GetRasterBand(1)` returns `nullptr` at this point.

Reply:
indeed. Would you mind issuing a PR against release/3.10 with this fix and the line at line 733 ?

Reply:
done, https://github.com/OSGeo/gdal/pull/11580